docs/src/rasters.md

```diff
@@ -118,3 +118,43 @@ end
 !!! note
 
     These methods are often used for reading/writing a block of image data efficiently, as it accesses "natural" blocks from the raster band without resampling, or data type conversion.
+
+# Using the DiskArray interface
+
+## Raster bands as 2D Disk Arrays
+
+As of ArchGDAL version 1.4.2 and higher a `RasterBand` is a subtype of `AbstractDiskArray` from the DiskArrays.jl package. This means that a `RasterBand` is also an `AbstractArray` and can therefore be treated like any Julia array. This means that square bracket indexing works in addition to the `read` methods described above.  
+
+````@example rasters
+band[1000:1010,300:310]
+````
+
+Also, windowed reading of the data can alternatively be done through the DiskArrays interface:
+
+````@example rasters
+using DiskArrays: eachchunk
+for (rows, cols) in take(eachchunk(band), 5)
+    @info "Window" rows, cols
+end
+````
+
+This code is equivalent to the window function mentioned in [Windowed Reads and Writes](@ref) but more portable because the raster band can be exchanged with any other type implementing the DiskArrays interface. Also, for many operations it will not be necessary anymore to implement the window loop, since the `DiskArrays` package provides efficient implementations for reductions and lazy broadcasting, so that for example operations like: 
```

Comment:
```suggestion
This code is similar to the window function mentioned in [Windowed Reads and Writes](@ref) but more portable because the raster band can be exchanged with any other type implementing the DiskArrays interface. Also, for many operations it will not be necessary anymore to implement the window loop, since the `DiskArrays` package provides efficient implementations for reductions and lazy broadcasting, so that for example operations like: 
```